### PR TITLE
fix: reply fix, chore: delete blog, fix:auth modal css

### PR DESCRIPTION
**Pull request by piyushagwl007**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#197 
#198 
#199 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by piyushagwl007** [comment]
Sign up modal fix
Sign in and sign up modal fix

**Comment by piyushagwl007** [comment]
Reply Bug fix
Reply Bug Fix

**Comment by piyushagwl007** [comment]
Delete blog fix
Delete Blog

File: client/src/components/deleteBlog/index.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <Modal.Description>
+        <p>You won&apos;t be able to undo this action.</p>
+        <Button color="red" 
+        disabled={deleting}
+        content="Confirm Delete"
+        icon="trash"
+         onClick={confirmDelete} 
+        loading={deleting}
+         />
+         {
```
Comment (nihalanisumit) [inline client/src/components/deleteBlog/index.jsx]: Add button as Modal.Action trust me it will look good. 2 line change.

Comment (piyushagwl007) [reply client/src/components/deleteBlog/index.jsx]: Okay wait

Comment (nihalanisumit) [reply client/src/components/deleteBlog/index.jsx]: https://react.semantic-ui.com/modules/modal/#types-controlled
check this

**Review by nihalanisumit — COMMENTED** [review]
Just update modal, small change :)

**Review by nihalanisumit — APPROVED** [review]
LGTM

**Comment by piyushagwl007** [comment]
Updated modal
delete

**Pull request merged**